src/main/java/org/tron/common/storage/DepositImpl.java

```diff
@@ -299,6 +291,7 @@ public synchronized void saveCode(byte[] codeHash, byte[] code) {
     return code;
   }
 
+  private static final int OLD_BLOCK_VERSION = 3;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move this variable to comply with Java Code Conventions. ([squid:S1213](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1213))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1667&issues=AWbODGoHLpqqSUlcKsP6&open=AWbODGoHLpqqSUlcKsP6)

[ISSUE_KEY]: AWbODGoHLpqqSUlcKsP6